### ISSUE TITLE
Ubuntu 20.04 - Node v12.18.0 - Text not rendering

### ISSUE THREAD

**Issue by lucasavila00** [tommywalkie/excalidraw-cli#1]
It will produce results with boxes and arrows but no text.
Changing font won't make it work.
File is valid and renders on excalidraw.com 

My system is:
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04 LTS"
NAME="Ubuntu"
VERSION="20.04 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```

Node: v12.18.0

NPM: 6.14.4

I'd like to help fix this, I might submit a PR if I can fix it.
I'd appreciate a a hint pointing where in the source code text rendering is done or any tips at all.

Thanks for the awesome job doing this :+1:

**Comment by tommywalkie** [comment]
Hi @lucasavila00, thanks for submitting the issue !
Unless it contains sensitive data, do you have the `.excalidraw` file you were using / an Excalidraw URL link so I can reproduce the issue ?

**Comment by lucasavila00** [comment]
Sure, 1 minute.

**Comment by lucasavila00** [comment]
This is the expected result (file was created in VSCode but opens in browser)
![Screenshot from 2020-06-25 11-15-04](https://user-images.githubusercontent.com/12750442/85736783-2b5c9b00-b6d5-11ea-8458-affd1781129f.png)

This is the output image:
![bug excalidraw](https://user-images.githubusercontent.com/12750442/85736251-b5f0ca80-b6d4-11ea-93c7-70716e26d584.png)

File:
https://gist.github.com/lucasavila00/faaddf118bdc85bcf20bf74098221daa

**Comment by lucasavila00** [comment]
I got it, kind of.

This file I sent [will not have a font size set for the element](https://github.com/tommywalkie/excalidraw-cli/blob/master/src/renderer.js#L284), which in turns make totalHeight be NaN and then[ y becomes NaN here](https://github.com/tommywalkie/excalidraw-cli/blob/master/src/renderer.js#L309).

**Comment by lucasavila00** [comment]
Let me know what you think of https://github.com/tommywalkie/excalidraw-cli/pull/3
The file I sent you now renders :)

**Comment by tommywalkie** [comment]
@lucasavila00 Found the issue. At the moment, `excalidraw-cli` only supports `fontSize` and `fontFamily` as font fields, these fields were apparently added in https://github.com/excalidraw/excalidraw/pull/1635, this might explain why I never encountered the `font` field until you reached me. Gotta look your PR.

**Comment by lucasavila00** [comment]
Thanks for stepping in so quickly to fix the issue :+1: 

I don't mind if you close my PR and do another one fixing it by the way, let me know if I can help with something else

**Comment by tommywalkie** [comment]
@lucasavila00 Thanks for the help ! Just need 1-2 hour and then I'll merge / add some fixes to your PR. 
By the way if you say your `.excalidraw` is working fine on Excalidraw itself, I suppose they didn't stop supporting `font` field after all.

**Comment by tommywalkie** [comment]
As mentionned in PR #3 :
> Merged #3 into master. Started working on `getFontByElement(...)` method in https://github.com/tommywalkie/excalidraw-cli/commit/6c94873d184195dacd0ae59941532dd81fad5179 so I can start optimizing the whole renderer codebase later. 👍 

Closing the issue. Thanks @lucasavila00 for the PR.

**Comment by lucasavila00** [comment]
Thanks :)
